src/tasks/RhythmTask.py

```
@@ -93,10 +93,10 @@ def do_run(self):
 
         while endless or count < total:
             count += 1
-            label = f"第 {count} 次" + ("" if endless else f"/{total}")
+            label = self.tr("第 {} 次").format(count) + ("" if endless else f"/{total}")
 
             # 点击开始演奏
-            self.log_info(f"{label}：点击开始演奏")
+            self.log_info(self.tr("{}：点击开始演奏").format(label))
```

Ruff (0.15.13)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)